atom/browser/api/atom_api_window.cc

```diff
@@ -858,6 +864,7 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("unhookWindowMessage", &Window::UnhookWindowMessage)
       .SetMethod("unhookAllWindowMessages", &Window::UnhookAllWindowMessages)
       .SetMethod("setThumbnailClip", &Window::SetThumbnailClip)
+      .SetMethod("setThumbnailTooltip", &Window::SetThumbnailTooltip)
```

Comment:
`setToolTip` sounds too generic, suggesting that it affect the window itself. It's a companion method to `setThumbnailClip`.